### PR TITLE
Add database model

### LINKED ISSUE #4: Create a database model
**As a** developer
**I need** database model for products
**So that** all the products can be stored and managed efficiently
   
 ### Details and Assumptions
 The products service will keep track of the products available for sale. 
We also need a unique identifier for each product. 
The database model must have the following attributes to keep track of at a minimum - 

- product_id
- product_name
- price
- description
- category

   
 ### Acceptance Criteria  
   
 ```gherkin
 Given the chosen database model
 When the products database is created
 Then the database should have all the required facilities and relationships to store the products information.
```

### PR DESCRIPTION
**Pull request by sahilbakshi98**
Added the basic initial database schema along with test cases

### CODE REVIEW

File: tests/factories.py
Comment (Leslielu7) [inline tests/factories.py:1]: Hey @sahilbakshi98,  a quick question, is "product3, product4, product5" in line 20 and "category3, category4, category5" in line 23 a design for testing a longer format of product name and category with symbols such as comma?

File: service/models.py
Comment (Alan-Ou) [inline service/models.py:1]: The code looks well-structured

File: tests/factories.py
Comment (Alan-Ou) [inline tests/factories.py:1]: It seems that the closing quotation mark is missing for some of the choices. For instance, product3 - product5, and category3 - category5.

Comment (sahilbakshi98) [reply tests/factories.py:1]: Hey Alan, good catch! Will fix that.

**Review by Leslielu7 — APPROVED** [review]

File: tests/factories.py
Comment (sahilbakshi98) [reply tests/factories.py:1]: Hey @Leslielu7, no, it was a typo on my end. I just intended to add dummy names for the products and categories in the factory generator.

**Pull request merged**